csaf_2.0/guidance/sbom-matching-interface.md

```diff
@@ -0,0 +1,91 @@
+|-------------------|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `vendor`          | `StringProperty`  | The value of the property is a string containing the vendor of the vulnerable product. Possible sources include:<br/>- Branches ([`branches_t`](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#312-branches-type)) in the CSAF advisory whose category is of type [`vendor`](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#3122-branches-type---category)<br/>- The `vendor` property of the CPE specified in the [`product_identification_helper`](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#3133-full-product-name-type---product-identification-helper) as well as of any CPE specified in the SBOM node<br/>- Any suitable property in the SBOM document indicating a vendor                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
To me the two leftmost columns read like a variabl-type declaration, where in code we would not expect so much text hanging off of the right side.

I can imagine a definition list or dedicated subsections with a nice introductory statemenet would be more readable.